website/docs/r/pki_secret_backend_acme_eab.html.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: "vault"
+page_title: "Vault: vault_pki_secret_backend_acme_eab resource"
+sidebar_current: "docs-vault-resource-pki-secret-backend-acme-eab"
+description: |-
+  Sets the EST configuration on a PKI Secret Backend for Vault.
```

Comment:
You may have overlooked updating this description.

Reply:
Fixed